recipes/graphviz/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+    ./configure --prefix=$PREFIX \
+                --with-quartz \
+                --disable-debug \
+                --disable-dependency-tracking \
+                --disable-java \
+                --disable-php \
+                --disable-perl \
+                --disable-tcl \
+                --without-x \
+                --without-qt
+else
+    ./configure --prefix=$PREFIX \
+                --disable-java \
+                --disable-php \
+                --disable-perl \
+                --disable-tcl \
+                --without-x \
+                --without-gtk
+fi
+
+make
+# This is failing for rtest.
+# Doesn't do anythoing for the rest
+# make check
```

Comment:
I prefer to leave this for the feedstock. I spent like an hour trying to come up with a patch for this but I couldn't do it :-)